app/src/main/java/com/techpaliyal/androidkotlinmvvm/ui/activity/MainActivity.kt

```diff
@@ -50,7 +50,7 @@ class MainActivity : AppCompatActivity() {
         }
 
         binding.btnMultipleViews.setOnClickListener {
-            MultipleViewTypeActivity.start(this)
```

Comment:
@IamRaja007  create new button and navigate to your desired activity

Reply:
Let me first brief what i have done here sir. Presently I am using this activity(the attached photo)  inside "Multiple View Types Test" button. Inside it, I have kept my new made activity and the School Listing activity that was already present.. I have done this to organize the examples pertaining to the same category sir.. Tell me, if you still want some changes.. I would implement them then. @yogeshpaliyal 
![WhatsApp Image 2021-10-05 at 4 11 15 PM](https://user-images.githubusercontent.com/71515810/136008282-6edd5fcb-e706-467c-86a4-b096f7fd989f.jpeg)

Reply:
Great, thanks for clearification